src/Suppressor.jl

```diff
@@ -1,14 +1,11 @@
 __precompile__()
 
 module Suppressor
-using Compat
 
 export @suppress, @suppress_out, @suppress_err
 export @capture_out, @capture_err
 export @color_output
 
-haslogging() = isdefined(Base, :CoreLogging)
```

Comment:
Julia 1+ always has `Base.CoreLogging`. The remainder of these changes are me removing the use of this conditional and un-indenting code.